### ISSUE TITLE
cURL error 60 when subscribing to a feed

### ISSUE THREAD

**Issue by ghost** [FreshRSS/FreshRSS#5122]
When I add a subscription source, cURL error 60: SSL certificate problem: unable to get local issuer certificate[ https://api.note-star.cn/api/feed/fetch ]And Unknown error for feed[ https://api.note-star.cn/api/feed/fetch ]What can I do to successfully subscribe? thank you.

**Comment by Frenzie** [comment]
```
$ curl -I https://api.note-star.cn/api/feed/fetch
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

Strangely, Firefox thinks it's fine.

**Comment by ghost** [comment]
I haven't had this problem on Chrome and edge

**Comment by Frenzie** [comment]
Yes, I meant Firefox didn't display any warnings about the certificate.

I forgot to say, but under advanced in add feed you can disable verification of certificates.

**Comment by ghost** [comment]
After I selected "Verify certificate security" as "No", a new problem occurred.
At first, the following error was reported: cURL error 59: failed setting cipher list: DEFAULT@SECLEVEL =1 [ https://api.note-star.cn/api/feed/fetch ]
The current error is: Unknown error for feed[ https://api.note-star.cn/api/feed/fetch ]

**Comment by Frenzie** [comment]
Ah, that's a duplicate of #3752. I didn't realize that hadn't been fixed yet. See https://github.com/FreshRSS/FreshRSS/issues/3752#issuecomment-895798061

If possible, using a newer version of OpenSSL would be the best solution.

**Comment by ghost** [comment]
I try to upgrade openssl to test whether it can be solved. thank you.
